tutorials/wpa-enterprise.rst

```diff
@@ -0,0 +1,373 @@
+
+.. image:: ../images/navigating-to-organization.png
+    :target: ../_images/navigating-to-organization.png
+
+From the organization's page, we need to find the organization's UUID and
+RADIUS token.
+
+.. image:: ../images/organization-uuid.png
+    :target: ../_images/organization-uuid.png
+
+.. image:: ../images/organization-radius-token.png
+    :target: ../_images/organization-radius-token.png
+
+This is good point to decide whether to use self-signed certificates or
+public certificates issued by a trusted Certificate Authority (CA). Both
+options have their pros and cons, and the choice largely depends on your
+specific requirements and constraints.
+
+Self-Signed Certificates
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pros:
+
+    - Generated locally without involving a third-party CA.
+    - Eliminates the need for external entities, reducing the risk of
+      compromised trust.
+
+Cons:
+
+    - Requires installation of self-signed CA on all client devices.
+
+Public Certificates
+~~~~~~~~~~~~~~~~~~~
+
+Pros:
+
+    - Issued by trusted CAs, thus works out of the box with most devices.
+
+Cons:
+
+    - Offers a higher risk of compromise.
+    - Cumbersome to set-up.
+
+We recommend to use Ansible OpenWISP2 role to which simplifies configuring
+FreeRADIUS to use WPA Enterprise. Please refer to the `"Configuring
+FreeRADIUS for WPA Enterprise (EAP-TTLS-PAP)" section in the
+ansible-openwisp2 documentation
+<https://github.com/openwisp/ansible-openwisp2/tree/master?tab=readme-ov-file#configuring-freeradius-for-wpa-enterprise-eap-ttls-pap>`_
+for details.
+
+If you still prefer to configure the FreeRADIUS site manually, you can
+refer the `"Freeradius Setup for WPA Enterprise (EAP-TTLS-PAP)
+authentication" section of the OpenWISP RADIUS documentation
+<https://openwisp-radius.readthedocs.io/en/stable/developer/freeradius_wpa_enterprise.html#freeradius-setup-for-wpa-enterprise-eap-ttls-pap-authentication>`_.
+
+Creating the NAS
+----------------
+
+.. note::
+
+    You can skip this step if you are following this tutorial on our
+    :doc:`Demo System <./demo>`. The NAS has been already configured on
+    the :doc:`Demo System <./demo>`.
+
+From the OpenWISP navigation menu, go to ``RADIUS`` and then ``NAS``, from
+here click on the ``Add NAS``.
+
+.. image:: ../images/navigating-to-nas.png
+    :target: ../_images/navigating-to-nas.png
+
+Fill in the organization, short name, secret, and set the type to
+"Wireless - IEEE 802.11". In the **name** field, enter the IP address of
+the NAS. Since every device acts as a NAS in our scenario, we specify the
+subnet of the VPN. This would allow FreeRADIUS to accept RADIUS traffic
+from all the devices.
+
+.. image:: ../images/wpa-enterprise/create-nas.png
+    :target: ../_images/create-nas.png
+
+.. warning::
+
+    Creating or modifying a NAS in OpenWISP requires a restart of the
+    FreeRADIUS server. Otherwise, the changes won't take effect.
```

Comment:
Let's clarify that this happens because FreeRADIUS reads the NAS from the DB only at startup.